drivers/adc/ad713x/ad713x.h

```diff
@@ -41,247 +42,292 @@
 #define SRC_AD713X_H_
 
 /******************************************************************************/
-/********************** Macros and Constants Definitions **********************/
+/***************************** Include Files **********************************/
 /******************************************************************************/
-/*
- * Create a contiguous bitmask starting at bit position @l and ending at
- * position @h.
- */
-#define GENMASK(h, l) \
-		(((~0UL) - (1UL << (l)) + 1) & (~0UL >> (31 - (h))))
 
+#include <stdbool.h>
+#include <stdio.h>
+#include "spi.h"
+#include "gpio.h"
+#include "util.h"
+
+/******************************************************************************/
+/********************** Macros and Constants Definitions **********************/
+/******************************************************************************/
 /*
  * AD713X registers definition
  */
 #define AD713X_REG_INTERFACE_CONFIG_A		0x00
 #define AD713X_REG_INTERFACE_CONFIG_B		0x01
-#define AD713X_REG_DEVICE_CONFIG		0x02
-#define AD713X_REG_CHIP_TYPE			0x03
-#define AD713X_REG_PRODUCT_ID_LSB		0x04
-#define AD713X_REG_PRODUCT_ID_MSB		0x05
-#define AD713X_REG_CHIP_GRADE			0x06
-#define AD713X_REG_SCTATCH_PAD			0x0A
-#define AD713X_REG_SPI_REVISION			0x0B
-#define AD713X_REG_VENDOR_ID_LSB		0x0C
-#define AD713X_REG_VENDOR_ID_MSB		0x0D
-#define AD713X_REG_STREAM_MODE			0x0E
+#define AD713X_REG_DEVICE_CONFIG			0x02
```

Comment:
It looks like there is a problem with indentation you use in your editor size of 4 spaces: 
size =4 -> https://github.com/analogdevicesinc/no-OS/blob/a6c6afb83168a0c32763397b8106376ecad64584/drivers/adc/ad713x/ad713x.h?ts=4
 
size =8 -> https://github.com/analogdevicesinc/no-OS/blob/a6c6afb83168a0c32763397b8106376ecad64584/drivers/adc/ad713x/ad713x.h?ts=8